stage1/init/common/pod.go

```diff
@@ -465,6 +465,10 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		unit.NewUnitOption("Service", "CapabilityBoundingSet", strings.Join(capabilitiesStr, " ")),
 	}
 
+	if ra.ReadOnlyRootFS {
+		opts = append(opts, unit.NewUnitOption("Service", "ReadOnlyDirectory", "/"))
```

Comment:
Indeed, spelling mistake.

I also wonder if we need to add [ReadWriteDirectories=](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ReadWriteDirectories=) for the RW volumes when we have a read-only rootfs.

More tests to add in tests/rkt_volume_test.go?